docs/user/collected/metrics.md

```diff
@@ -0,0 +1,63 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+
+# Metrics
+This document enumerates the metrics collected by this project.
+This project may depend on other projects which also collect metrics.
+This means you might have to go searching through the dependency tree to get a full picture of everything collected by this project.
+Sorry about that.
```

Comment:
I kinda feel like this "Sorry about that" sounds weird here (I know, it's included in the Glean template, so I should take the discussion there)

Reply:
I can file a bug about this.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1586730